packages/components/src/components/input/input.ts

```diff
@@ -279,6 +280,13 @@ export default class SdInput extends SolidElement implements SolidFormControl {
     event.stopPropagation();
   }
 
+  private handleSearchClick(event: MouseEvent) {
+    this.value = '';
```

Comment:
As discussed on call, please remove this line so that users can decide if the input value should be cleared or not